living_documentation_regime/living_documentation_generator.py

```diff
@@ -308,14 +310,16 @@ def _generate_markdown_pages(self, issues: dict[str, ConsolidatedIssue]) -> None
         # Generate a markdown page for every issue
         for consolidated_issue in issues.values():
             self._generate_md_issue_page(issue_page_detail_template, consolidated_issue)
-            if consolidated_issue.errors:
-                repository_id: str = consolidated_issue.repository_id
-                number: int = consolidated_issue.number
-                html_url: str = consolidated_issue.html_url
-                for error_type, error_message in consolidated_issue.errors.items():
-                    report_page_content += (
-                        f"| {error_type} | [{repository_id}#{number}]({html_url}) | {error_message} |\n"
-                    )
+            if is_report_page:
```

Comment:
Connect to one if line?

Reply:
Solved at: [21d40cf](https://github.com/AbsaOSS/living-doc-generator/pull/79/commits/21d40cf926291b07986ba03f20ef4a619bc72aed).